.kokoro/build.sh

```diff
@@ -42,6 +42,11 @@ elif [ "$JOB_TYPE" = "continuous" ]; then
         rbenv global "$version"
         (bundle update && bundle exec rake kokoro:continuous) || set_failed_status
     done
+elif [ "$JOB_TYPE"] = "samples_presubmit" ]; then
+    for version in "${versions[@]}"; do
```

Comment:
I think we might want to consider running this against just one Ruby version (maybe the latest), to limit the tax on the infrastructure.